.github/workflows/build.yml

```diff
@@ -5,9 +5,9 @@ jobs:
     strategy:
       matrix:
         java: [17, 18]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Why change this? Changing it from latest to something different just ensures we have to change it again later down the line.

Reply:
I had this to 22.04 which included the quilt-loader project and others back when 22.04 wasn't marked in "latest" context meaning it had used 20.04 LTS at time of that commit, but will undo it now.